tools/release.js

```diff
@@ -395,7 +395,7 @@ const run = async () => {
       colors.info(" Making sure the generated version is ok... 🕵🏻")
     );
     await node(
-      `./tools/respec2html.js -e --timeout 30 --src file://${process.cwd()}/examples/basic.built.html --out /dev/null`,
+      `./tools/respec2html.js -e --timeout 30 --src file:///${__dirname}/../basic.built.html --out /dev/null`,
```

Comment:
I accidentally missed `../` part, but still think `__dirname` is better because it's cwd-safe. What do you think? @marcoscaceres

Reply:
I'm ok with __dirname, but it was breaking. I usually run the release script from:

```Bash
./tools/release.js
``` 

Thus need to make sure that it always resolves to the right place.